store/src/main/java/org/apache/rocketmq/store/CommitLog.java

```diff
@@ -1179,10 +1222,11 @@ public long getJointime() {
 
         private final StringBuilder msgIdBuilder = new StringBuilder();
 
-        private final ByteBuffer hostHolder = ByteBuffer.allocate(8);
+//        private final ByteBuffer hostHolder = ByteBuffer.allocate(8);
```

Comment:
If the code is no use, just delete them may be better.

Reply:
done!